schemas/1.0/application.json

```diff
@@ -97,13 +98,15 @@
     "items": { "$ref": "#/definitions/document" }
   },
   "additional_information": { "type": ["string", "null"] },
+  "has_partner": { "type": ["string", "null"] },
```

Comment:
I think 'has_partner' should be moved into client_details or or applicant.

Reply:
Make enum yes no